index.html

```diff
@@ -819,9 +819,8 @@ <h2>HTTP DELETE</h2>
         <h2>Version Containers (<a>LDPCv</a>)</h2>
         <p>
           An <a>LDPCv</a> is both a <a>TimeMap</a> per
-          Memento [[!RFC7089]] and an <a>LDPC</a>. As a <a>TimeMap</a> an <a>LDPCv</a> MUST conform to the
-          specification for such resources in [[!RFC7089]]. An implementation MUST indicate <a>TimeMap</a> in the
-          same way it indicates the container interaction model of the resource via HTTP headers.
+          Memento [[!RFC7089]] and an <a>LDPC</a>. As a <a>TimeMap</a>, an <a>LDPCv</a> MUST conform to
+          Memento [[!RFC7089]].
         </p>
         <p>
           An implementation MUST NOT allow the creation of an <a>LDPCv</a> that is <a>LDP-contained</a> by its
```

Comment:
All uses of `An implementation` should be changed to `Implementations`. See [bullet no.2](https://docs.google.com/document/d/1n7ZVIRaiYG36Ye0AP4oSTYbDN915box6zVteXciojFc/edit?usp=sharing).